charts/athens-proxy/templates/deployment.yaml

```diff
@@ -24,6 +24,9 @@ spec:
         checksum/upstream: {{ include (print $.Template.BasePath "/config-upstream.yaml") . | sha256sum }}
         checksum/ssh-config: {{ include (print $.Template.BasePath "/config-ssh-git-servers.yaml") . | sha256sum }}
         checksum/ssh-secret: {{ include (print $.Template.BasePath "/secret-ssh-git-servers.yaml") . | sha256sum }}
+        {{- if .Values.storage.s3.iamRole }}
+        iam.amazonaws.com/role: {{ .Values.storage.s3.iamRole | quote }}
```

Comment:
@aaqel-s will a controller/operator need to be present in the cluster to detect this annotation?

Reply:
Yup, here's an example https://github.com/uswitch/kiam